lib/data/datasource/remote/api_service.dart

```diff
@@ -87,4 +90,39 @@ class ApiService {
       rethrow;
     }
   }
+
+  Future<DataState<List<MovieModel>>> getFavoriteMovies(
```

Comment:
I'd rename this as `getMoviesByIds` to have a more generic method. Also, `getFavoriteMovies` gives a clue that those favorites are hosted in the API, which is not the case

Reply:
done